### DISCUSSION TITLE
Supported devices list request.

### DISCUSSION THREAD

**Discussion by esselfe** [NVIDIA/open-gpu-kernel-modules#185] [Ideas]
Hi, a list of the supported device models would be appreciated.

**Comment by ElijahPepe** [comment]
Only [Turing](https://en.wikipedia.org/wiki/Turing_(microarchitecture)#Products_using_Turing) and [Ampere](https://en.wikipedia.org/wiki/Ampere_(microarchitecture)#Products_using_Ampere) GPUs are supported IIRC (for most consumers, the RTX 30X series, the RTX 20X series, and the GTX 16X series).

**Reply by esselfe** [reply]
Alright, I have a Quadro 600 and a GTX 750ti, can't wait to see those working :)

**Reply by MartinX3** [reply]
It will never happen.

**Comment by mtijanic** [comment]
More info and discussions about such a list at https://github.com/NVIDIA/open-gpu-kernel-modules/issues/18
